tests/contrib/test_keyring_storage.py

```diff
@@ -68,7 +65,8 @@ def test_locked_get(self):
                                return_value=mock_content,
                                autospec=True) as get_password:
             with mock.patch(
-                    'oauth2client.keyring_storage.Credentials') as MockCreds:
+                'oauth2client.contrib.keyring_storage.Credentials') \
```

Comment:
There's no other way to wrap this. You can't do `with (expr as value)`, it's inexplicably invalid syntax.